nameserver/nameserver.go

```diff
@@ -129,10 +231,13 @@ func (n *Nameserver) ContainerDied(ident string) {
 	entries := n.entries.tombstone(n.ourName, func(e *Entry) bool {
 		if e.ContainerID == ident {
 			n.infof("container %s died; tombstoning entry %s", ident, e.String())
+			// We want to restore all entries if container comes back
+			e.stopped = true
```

Comment:
I'm struggling to see that at https://github.com/weaveworks/weave/pull/2199/files#diff-079f8b38f0b2e637d5cd2138cb73ed4fR200